dist/services/iiif-parser.js

```diff
@@ -118,13 +118,13 @@ function getChoiceItems(manifest) {
 function getLabelValue(label) {
   if (label && (0, _typeof2["default"])(label) === 'object') {
     // English
-    if (label.hasOwnProperty('@en')) {
-      return label['@en'].length > 0 ? label['@en'][0] : '';
+    if (label.hasOwnProperty('en')) {
```

Comment:
Can you explain the change from @ to non-@?

Reply:
Yes it's a iiif spec change.